assemblies/pentaho-solutions/src/main/resources/pentaho-solutions/system/applicationContext-spring-security-oauth.properties

```diff
@@ -0,0 +1,20 @@
+shouldPerformLiveUpdate=true
+
+# OIDC Configuration for Azure AD
+azure.id=azure
+azure.client-name=azure
+azure.client-id=8ca223e9-fc41-4751-8dc3-7b1fbdabcbb3
+azure.client-secret=Anything
+azure.scope=openid,profile,email,User.ReadBasic.All,RoleManagement.Read.Directory
+azure.redirect-uri=http://localhost:8080/pentaho/login/oauth2/code/azure
+azure.token-uri=https://login.microsoftonline.com/18791e17-6159-4f52-a8d4-de814ca8284a/oauth2/v2.0/token
```

Comment:
This might be along the lines of Luc earlier comment https://github.com/pentaho/pentaho-platform/pull/5879/files#r2052620454

is the string "18791e17-6159-4f52-a8d4-de814ca8284a" an actual account tenant ~ guid ?

- https://learn.microsoft.com/en-us/entra/identity-platform/v2-oauth2-auth-code-flow

If so, are we putting into production an actual account tenant, I would think we would use some obvious string to denote a customer needs to configure it

Microsoft uses `{tenant}` for example
```https://login.microsoftonline.com/{tenant}/oauth2/v2.0/.....```